SECURITY.md

```diff
@@ -0,0 +1,24 @@
+## Report a Vulnerability
+
+We're extremely grateful for security researchers and users that report
+vulnerabilities to the CRI-O community. All reports are thoroughly investigated
+by a set of community volunteers.
+
+To make a report, email your vulnerability to the private
+[cncf-crio-security@lists.cncf.io](mailto:cncf-crio-security@lists.cncf.io) list
+with the security details and the details expected for [all CRI-O bug
+reports](.github/ISSUE_TEMPLATE/bug-report.yml).
```

Comment:
I think it would be better to point them at the raw file for ease in cut/paste into email:  
https://raw.githubusercontent.com/cri-o/cri-o/0eca48b4f067561db2530506a72d9ea6d7a0dd47/.github/ISSUE_TEMPLATE/bug-report.yml.  YMMV

```suggestion
reports](.github/ISSUE_TEMPLATE/bug-report.yml).
```

Reply:
Sure, fixed as suggested :+1: